discover/discover.go

```diff
@@ -26,6 +26,7 @@ type Options struct {
 	Namespace   string
 	Clustername string
 	Labels      string
+	Fromsource  string
```

Comment:
FromSource  string

Reply:
The naming convention is **Fromsource**. 
Need to use the same.

Reply:
I would prefer both of em to be updated to `FromSource` unless there's some other limitation

Reply:
> But we are maintaining this naming convention for all other parameters as well

Only in the worker protobuf in which these fields were added recently. In general the type system in autopol seems to be camelCase only. But yeah it's fine if we want to keep following this.